daal4py/sklearn/cluster/_dbscan_0_21.py

```diff
@@ -43,10 +43,12 @@ def _daal_dbscan(X, eps=0.5, min_samples=5, sample_weight=None):
     XX = make2d(X)
 
     fpt = getFPType(XX)
+    memorySavingMode = True if is_in_sycl_ctxt() else False
```

Comment:
It might be risky to set memorySaavingMode to false on CPU in all cases.  How to process huge datasets?

Reply:
We don't support memorySaavingMode param in Sklearn interface at this moment. But we can extend it in future. 'False' value is default for DAAL itself. You can create an issue/task about it.

Reply:
more explicit `memorySavingMode = bool(is_in_sycl_ctxt())`.

Reply:
I think that current representation is better because it is easier to read.
"memorySavingMode is True because we are in sycl context" but not "memorySavingMode is equal to is_in_sycl_context()".
Imagine any non-bool parameter that should be dependent on the context.
As example: method = 'specialSyclMethod' if is_in_sycl_ctxt() else 'defaultDense'